src/main/java/uk/ac/ebi/eva/pipeline/io/mappers/VariantAggregatedVcfFactory.java

```diff
@@ -418,5 +419,41 @@ protected void addGenotypeWithGTS(Variant variant, VariantSourceEntry sourceEntr
             }
         }
     }
+
+    protected void checkVariantInformation(Variant variant, String fileId, String studyId) throws NonVariantException, IncompleteInformationException {
```

Comment:
Part of this method is duplicated from the `checkVariantInformation` method in `VariantVcfFactory` and we are inheriting from that class

Reply:
As discussed, the second part of the method is specific for aggregated types job and can't be moved to VariantVcfFactory as it has code speicfic for genotype job.